downstairs/src/lib.rs

```diff
@@ -1736,12 +1944,127 @@ impl Downstairs {
                 };
 
                 Ok(Some(Message::FlushAck {
+                    upstairs_id: job.upstairs_connection.upstairs_id,
+                    session_id: job.upstairs_connection.session_id,
+                    job_id,
+                    result,
+                }))
+            }
+            IOop::ExtentClose {
+                dependencies: _,
+                extent,
+            } => {
+                println!("Closing extent {}", extent);
```

Comment:
slog instead?

Reply:
Fixed in 02d694383914e8a615080888c1b1bcf9de1be8a3